src/privileged/FirefoxMonitor.css

```diff
@@ -0,0 +1,85 @@
+#fxmonitor-notification .popup-notification-body {
+  font-size: 110%;
+}
+
+#fxmonitor-notification popupnotificationcontent {
+  margin-top: 0;
+}
+
+#fxmonitor-notification .popup-notification-body > hbox:first-child {
+  display: none;
```

Comment:
That feels a little unstable, but I understand there's no better way to do it :/

Reply:
There probably is, I'll try.